ping-protect/src/main/java/org/forgerock/android/auth/PingOneProtectEvaluationCallback.kt

```diff
@@ -8,21 +8,19 @@ package org.forgerock.android.auth
 
 import android.content.Context
 import androidx.annotation.Keep
-import org.forgerock.android.auth.callback.AbstractCallback
+import org.forgerock.android.auth.callback.AbstractProtectCallback
+import org.forgerock.android.auth.callback.HiddenValueCallback
+import org.forgerock.android.auth.callback.PING_ONE_RISK_EVALUATION_SIGNALS
 import org.json.JSONObject
 
 private val TAG = PingOneProtectEvaluationCallback::class.java.simpleName
 
 /**
  * Callback to evaluate Ping One Protect
  */
-open class PingOneProtectEvaluationCallback : AbstractCallback {
+open class PingOneProtectEvaluationCallback @Keep constructor(jsonObject: JSONObject, index: Int) :
```

Comment:
We have to use @keep for both constructors, currently, we use reflection to construct the class. You may want to enable 
`    isMinifyEnabled = true
    isShrinkResources = true`
in the AndroidBuildGradlePlugin, build a release build, install the apk.

Reply:
done as part of Jey's comments